src/watcher.js

```diff
@@ -143,7 +143,7 @@ function Watcher(bot, league) {
             self.bot.say({
                 text: "<@" + white + ">,  <@" + black + ">:"
                     + " Your game is *not valid* because "
-                    + "*" + result.reason + "*",
+                    + "*" + result.reason + "*" + "@seb32",
```

Comment:
The linter is complaining about this line.

It was you to remove the unnecessary concatenation.
 
`"...*"+"@seb32"`

Replace with a single string constant.